src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/ReinforcedChestBreak.java

```diff
@@ -0,0 +1,110 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.SimpleHack;
+import com.programmerdan.minecraft.simpleadminhacks.configs.ReinforcedChestBreakConfig;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockBreakEvent;
+import vg.civcraft.mc.citadel.Citadel;
+import vg.civcraft.mc.citadel.ReinforcementManager;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Sends every 3 minutes a message to the admins if a chest is broken
+ */
+public class ReinforcedChestBreak extends SimpleHack<ReinforcedChestBreakConfig> implements Listener {
+
+    public static String NAME = "ReinforcedChestBreak";
+
+    private ReinforcementManager manager;
+    private Messenger messenger;
+    private List<String> messages;
+
+    public ReinforcedChestBreak(SimpleAdminHacks plugin, ReinforcedChestBreakConfig config) {
+        super(plugin, config);
+    }
+
+    @Override
+    public void registerListeners() {
+        Bukkit.getPluginManager().registerEvents(this, plugin());
+    }
+
+    @Override
+    public void registerCommands() {}
+
+    @Override
+    public void dataBootstrap() {
+        messages = new ArrayList<>();
+
+        manager = Citadel.getReinforcementManager();
+        messenger = new Messenger();
+
+        Bukkit.getScheduler().runTaskTimer(plugin(), messenger, 0, config.getDelay() * 20);
+    }
+
+    @Override
+    public void unregisterListeners() {}
+
+    @Override
+    public void unregisterCommands() {}
+
+    @Override
+    public void dataCleanup() {}
+
+    @Override
+    public String status() {
+        return "Delay: " + config.getDelay();
+    }
+
+    /**
+     * Gets fired by the BLockBreakEvent and checks if the block is reinforced
+     * @param eve BLockBreakEvent
+     */
+    @EventHandler
```

Comment:
> So in current incarnation every attempted break of a reinforced block will queue up a message, and players can easily make 3-5 break attempts per second, depending on the block.

AFAIK the BlockBreakEvent is only called if a block is destroyed, not if someone is trying to break a block? Or is this a problem because Citadel cancels the event?